### ISSUE TITLE
Error occurred when loading some widgets

### ISSUE THREAD

**Issue by ambience-kim** [FlutterFlow/flutterflow-issues#2632]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

![Slide1](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/121ad7cc-56ad-4057-a992-0d323c8ba8e2/9dfc9b33-94cd-4ec4-a117-6d2374807181?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8xMjFhZDdjYy01NmFkLTQwNTctYTk5Mi0wZDMyM2M4YmE4ZTIvOWRmYzliMzMtOTRjZC00ZWM0LWExMTctNmQyMzc0ODA3MTgxIiwiaWF0IjoxNzExOTc4NDMzLCJleHAiOjMzMjgyNTM4NDMzfQ.RWrndRaudHP_8Vh3HGwTtHXB1odNosZlSrxVsBM90Bo)

![Slide2](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/2bf684e1-3df6-4853-96e9-33d22b80b896/075a6a3a-6494-4b0a-9f45-d52e456a9519?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8yYmY2ODRlMS0zZGY2LTQ4NTMtOTZlOS0zM2QyMmI4MGI4OTYvMDc1YTZhM2EtNjQ5NC00YjBhLTlmNDUtZDUyZTQ1NmE5NTE5IiwiaWF0IjoxNzExOTc4NDMzLCJleHAiOjMzMjgyNTM4NDMzfQ.QoajUR4n7Tm8tpanp-z0VoSFj85i9Lac5JsL7UUaPn0)

![Slide3](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/541aa538-06e0-4ac9-b82b-54d84e34efce/47158ded-583f-485f-b380-b0e425f04413?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS81NDFhYTUzOC0wNmUwLTRhYzktYjgyYi01NGQ4NGUzNGVmY2UvNDcxNThkZWQtNTgzZi00ODVmLWIzODAtYjBlNDI1ZjA0NDEzIiwiaWF0IjoxNzExOTc4NDMzLCJleHAiOjMzMjgyNTM4NDMzfQ.xxWvxL1wUP_vV3TdfsZ48ixgI1eN6-Q9SrZqazHs5ZU)

![Slide4](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/7c657fbb-3725-4b79-8d0d-2d4e8878560c/b2308371-eccc-44bf-a9ba-9b8cc94e512d?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS83YzY1N2ZiYi0zNzI1LTRiNzktOGQwZC0yZDRlODg3ODU2MGMvYjIzMDgzNzEtZWNjYy00NGJmLWE5YmEtOWI4Y2M5NGU1MTJkIiwiaWF0IjoxNzExOTc4NDMzLCJleHAiOjMzMjgyNTM4NDMzfQ.JUKTKkV5gfxKiFk_qe_LXcZfBA7b3eIrHasFtvqu0OI)

![Slide5](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/384a5c97-bbac-4d5f-9dc7-3374961ac4bf/b2ef3114-50b5-499e-8d12-7da75a29ab76?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS8zODRhNWM5Ny1iYmFjLTRkNWYtOWRjNy0zMzc0OTYxYWM0YmYvYjJlZjMxMTQtNTBiNS00OTllLThkMTItN2RhNzVhMjlhYjc2IiwiaWF0IjoxNzExOTc4NDMzLCJleHAiOjMzMjgyNTM4NDMzfQ.OWFR9wWVBepwRbO_NlyxrcHYFEngdLqeoXZ3Ue9oejQ)

![Slide6](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/f7283290-967d-4beb-b617-1766e13fb606/90498af0-1a43-421b-819f-4d77642eb4e3?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9mNzI4MzI5MC05NjdkLTRiZWItYjYxNy0xNzY2ZTEzZmI2MDYvOTA0OThhZjAtMWE0My00MjFiLTgxOWYtNGQ3NzY0MmViNGUzIiwiaWF0IjoxNzExOTc4NDMzLCJleHAiOjMzMjgyNTM4NDMzfQ.UxXlrC0cr2erfHFBUm_o6lpQRLDe-_z5deo7mfgEyss)

[https://github.com/FlutterFlow/flutterflow-issues/assets/140410622/4d9ac5e9-9ef0-4647-9968-f99da5424127](https://github.com/FlutterFlow/flutterflow-issues/assets/140410622/4d9ac5e9-9ef0-4647-9968-f99da5424127)

### Expected Behavior

No errors should occur

### Steps to Reproduce

Described in detail in previous entries.

Because there was a problem with the loading of pages that were operating normally, it is impossible to reproduce in the blank project below.

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4Oz8jc34VIsdtb+KrcKMJVnjw5JlgjbbxM08FHd0o3F7nNG4gIZfTBXE9uOP25dglcPk39hjkfw8LOm9z1Nu0rCyqbf6Fz1LlQehbLXj26M66YEsywPmhEGshnIFCiyqSNvx1AHO5pSE4D3FyMOfPLcDLvO7L1NTAFDfm1Kp/XrgfuQ0SLb3kNh1JWeC/v

### Context

Very serious.
I can't work because of this problem.

### Visual documentation

[Error_20240401.pdf](https://github.com/FlutterFlow/flutterflow-issues/files/14819468/Error_20240401.pdf)

### Additional Info

[https://github.com/FlutterFlow/flutterflow-issues/assets/140410622/45c88024-39a7-407b-a28a-0b57999a32c1](https://github.com/FlutterFlow/flutterflow-issues/assets/140410622/45c88024-39a7-407b-a28a-0b57999a32c1)

### Environment

```markdown
- FlutterFlow version: 3.19.1
- Platform: MacOS
- Browser name and version: Desktop Apps (MacOS)
- Operating system and version affected: MacOS (Venture) V13.6.8
```

**Comment by captainjdi** [comment]
@bozzoleandro I've been seeing similar issues with the TabBar in one of my projects. I created a new project to try to troubleshoot it, and I discovered the following steps to reproduce at least one variant of the issue. 

1. Create a new project
2. Add a new, blank component
3. Add a TabBar widget as a child to the container widget that appears on the component you just created
4. Click on the HomePage that was created with the new project
5. Click on the new component you created
6. Click on the 2nd tab in the TabBar

Result: Breaking crash error
<img width="1277" alt="Screen Shot 2024-04-09 at 4 18 00 PM" src="https://github.com/FlutterFlow/flutterflow-issues/assets/133725139/195ed644-3f41-48ae-9de7-b44b046d4acc">

Interestingly, if you click on the 2nd tab in the TabBar before clicking on the HomePage, the error doesn't seem to occur. If you then exit the project, go back in, and start clicking on the tabs, you'll get an error later though. 
<img width="1269" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/133725139/d3ec3847-2ea0-40a5-bd88-ccab4805fd5b">
